### PR TITLE
Fix .input-group sizing in IE11

### PR DESCRIPTION
**Pull request by zlima**
On IE11 input-group forms looks like this:

screenshot_3

Since line-height doesn't work right on IE11 we've used this fix to resolve the problem on the input size inside a form. 

We've change the $input-height calculation to consider the input-border-width and used the mixin already created for the input-size.

the final result looks like this: 
screenshot_6

We've tested on IE and chrome(windows, os x and ios) all the browsers with 100% zoom and it works, it also passed on Travis-ci test

### CODE REVIEW

**Comment by wolfy1339** [comment]
Please exclude files in dist/ from Pull Requests

**Comment by cvrebert** [comment]
Presumably fixes #17890/#18083

File: scss/_variables.scss [hunk 1]
```diff
@@ -79,6 +79,7 @@ $spacers: (
   )
 ) !default;
 $border-width:               1px !default;
+$input-border-width:         .0625rem !default;
```
Comment (cvrebert) [inline scss/_variables.scss]: Use the existing (well, it's in the latest `v4-dev`) `$input-btn-border-width` variable instead.

**Comment by cvrebert** [comment]
Also, please revert the change to `grunt/change-version.js`'s mode/permissions.

File: scss/_forms.scss [hunk 2]
```diff
@@ -60,7 +59,7 @@
     cursor: $cursor-disabled;
   }
 }
-
+@include input-size(".form-control",$input-height, $input-padding-y, $input-padding-x, $font-size-base, $line-height, $input-border-radius);
```
Comment (cvrebert) [inline scss/_forms.scss]: Add a space after the 1st comma

File: scss/_variables.scss (hunk 1 above)
Comment (zlima) [inline scss/_variables.scss]: the problem is that  **$input-heigth** calculation uses "rem" units and **$input-btn-border-width** uses "px", i could change to **$input-border-width:         rem-calc(1) !default;** i could also make the calculation convert on the **$input-heigth** calculation formula.
Which one is the best in this situation?  Or do you have any suggestion?

**Comment by mdo** [comment]
This was resolved with the move to flexbox.

**Pull request closed without merging**